python/whylogs/api/writer/whylabs.py

```diff
@@ -32,6 +32,9 @@
 )
 
 FIVE_MINUTES_IN_SECONDS = 60 * 5
+DAY_IN_SECONDS = 60 * 60 * 24
+WEEK_IN_SECONDS = DAY_IN_SECONDS * 7
+FIVE_YEARS_IN_SECONDS = DAY_IN_SECONDS * 365 * 5
```

Comment:
leap years :)

Reply:
I thought the same thing and decided to keep this simplistic for a hint/warning.